seeders/studentWorkProjects.seeder.js

```diff
@@ -12,27 +12,27 @@ const data = async () => {
   return [
     {
       name: "Prog1 Assignment 1",
```

Comment:
Same as above, we should circle back on the other PR to make sure we're spelling things out fully.

Reply:
This will be updated in the next push.